Asset.php

```diff
@@ -17,21 +17,28 @@ class Asset extends AssetBundle
     /**
      * @inheritdoc
      */
-    public $sourcePath = '@vova07/select2/assets';
+    public $sourcePath = '@vova07/select2/assets/select2-3.5.0';
```

Comment:
I'm not sure if this is right path. If we'll use `bower-asset` than it may be something like this: `@bower/select2`. Did you tested it?